### PR TITLE
Add banner for NS billing date

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
Add a similar messaging as Outer in the form of a banner to indicate the March 1 N&S billing update.

# Reasons
We've made a lot of changes to the docs about N&S costs and those changes are not GA and have been delayed.

### CODE REVIEW

**Comment by devinmarieb** [comment]
I forgot we mostly agreed on "web app" and my brain wanted to write "web application" so I had to go back and change the wording which is why this is three separate commits 🙃

**Comment by devinmarieb** [comment]
Had a real brain fart when making these changes. I updated the instances of "web application" to "web app" remembering that's what we agreed on. And I found an error (forgot to delete the word "date" from the sentence) after I pushed.

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -298,7 +298,10 @@ You can choose to pay for premium features per active user, compute, and optiona
 {: #how-do-I-calculate-my-monthly-storage-and-network-costs }
 {:.no_toc}
 
-Calculate your monthly storage and network costs by finding your storage and network usage on the [CircleCI app](https://app.circleci.com/) by navigating to Plan > Plan Usage.
+**NOTE:** Billing for network egress and storage will start to take effect on **March 1 2022** (subject to change). CircleCI is adding variables and controls to help you manage network and storage usage. The information in this section is applicable after the changes take effect on March 1, 2022. Current usage can be found on the [CircleCI web app](https://app.circleci.com/) by navigating to **Plan > Plan Usage**.
+{: class="alert alert-info" }
+
+Calculate your monthly storage and network costs by finding your storage and network usage on the [CircleCI web app](https://app.circleci.com/) by navigating to Plan > Plan Usage.
```
Comment (rosieyohannan) [inline jekyll/_cci2/faq.md:304]:
```suggestion
Calculate your monthly storage and network costs by finding your storage and network usage on the [CircleCI web app](https://app.circleci.com/) by navigating to **Plan > Plan** Usage.
```

**Review by rosieyohannan — APPROVED** [review]
Looks great! just one formatting thing :-D

**Pull request merged**